core/src/main/java/com/linecorp/armeria/common/stream/DeferredStreamMessage.java

```diff
@@ -35,6 +35,9 @@
  */
 public class DeferredStreamMessage<T> implements StreamMessage<T> {
 
+    private static final PendingSubscription<Object> NO_OP_PENDING_SUBSCRIPTION =
```

Comment:
Maybe `NO_OP_ALREADY_SUBSCRIBED` ? As far as I can tell, it's not really pending after this is set, it's more of a marker that there won't be anymore subscriptions.